index.html

```diff
@@ -58,5 +58,224 @@ <h3>LET'S CONNECT</h3>
+                  </label>
+                </summary>
+
+                <ul class="tech">
+                  <li class="techh">
+                    <img src="Images/Javascript.svg" alt="" />
+                    <span class="tech__">JavaScript</span>
+                  </li>
+                  <li class="techh">
+                    <img src="Images/html.svg" alt="" />
+                    <span class="tech__">HTML</span>
+                  </li>
+                  <li class="techh">
+                    <img src="Images/css.png" alt="" />
+                    <span class="tech__">css</span>
+                  </li>
+                </ul>
+                <div class="summary-down">
+                  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-up"><polyline points="18 15 12 9 6 15"></polyline></svg>
+                </div>
+              </details>
+              <details>
+                <summary>FrameWorks
+
+                  <label class="summary-up">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-down"><polyline points="6 9 12 15 18 9"></polyline></svg>
+                  </label>
+                </summary>
+                <div class="summary-down">
+                  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-up"><polyline points="18 15 12 9 6 15"></polyline></svg>
+                  </div>
+              </details>
+              <details>
+                <summary>Skills
+
+                  <label class="summary-up">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-down"><polyline points="6 9 12 15 18 9"></polyline></svg>
+                  </label>
+                </summary>
+                <div class="summary-down">
+                  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-up"><polyline points="18 15 12 9 6 15"></polyline></svg>
+                  </div>
+              </details>
```

Comment:
- [ ] A list would be more appropriate to group the 'Languages', 'Frameworks' and 'Skills'. Kindly fix that.

Reply:
why do you want me to put it on a list? 
I am using a details tag with a summary, I think it would be more useful in this case.
thank you